docs/source/example_input/boosted_frame_script.py

```diff
@@ -137,19 +137,26 @@ def dens_func( z, r ):
 # Velocity of the Galilean frame (for suppression of the NCI)
 v_comoving = - c * np.sqrt( 1. - 1./boost.gamma0**2 )
 
-# The diagnostics
-diag_period = 50        # Period of the diagnostics in number of timesteps
-# Whether to write the fields in the lab frame
-Ntot_snapshot_lab = 15
-dt_snapshot_lab = 10*(zmax-zmin)/c
-track_bunch = False  # Whether to tag and track the particles of the bunch
-
 # The interaction length of the simulation (meters)
 L_interact = (p_zmax-p_zmin) # the plasma length
 # Interaction time (seconds) (to calculate number of PIC iterations)
 T_interact = boost.interaction_time( L_interact, (zmax-zmin), v_window )
 # (i.e. the time it takes for the moving window to slide across the plasma)
 
+## The diagnostics
```

Comment:
Here, I could imagine something like this (I did not test the code - this is open for discussion):

```
# Number of diagnostics to output (including first output at t=0)
N_diag = 10+1
# Time interval between diagnostics (first at t=0, last at t=T_interact)
dt_diag = (L_interact + (zmax-zmin)) / v_window / (N_diag - 1)
# Period of the (boosted frame) diagnostics in timesteps
diag_period = T_interact / (N_diag - 1)
# Period of writing the cached, backtransformed lab frame diagnostics to disk 
write_period = 50
```

This would ensure that in the case of gamma_boost=1, the actual standard diagnostics are defined at the same times/positions as the lab frame output of any boosted frame simulation. (Makes it easy to compare results between lab and boosted frame runs).

The boosted frame diagnostics would always be equally distributed within the boosted interaction time.

If we agree to this (or something similar), I would suggest to change the other two example scripts accordingly.

Reply:
done